### DISCUSSION TITLE
Problemas para exportar archivo XLS

### DISCUSSION THREAD

**Discussion by locurandres** [chamilo/chamilo-lms#5469] [General]
Hola necesito ayuda, cuando intento generar un **informe de Seguimiento de exámenes** me larga la siguiente pantalla de error, alguien podría guiarme para poder solucionarlo? gracias.
![error00](https://github.com/chamilo/chamilo-lms/assets/74663717/11d0d331-fcbe-4110-8885-21bcadc98ee4)
![error01](https://github.com/chamilo/chamilo-lms/assets/74663717/7e099692-6d2e-4516-9cea-3524ea72561b)

**Comment by nosolored** [comment]
Hola
Ese error apunta a que estás utilizando una versión de PHP que no es totalmente compatible con tu versión de Chamilo.
Si utilizas Chamilo 1.11.x, ten en cuenta que tiene compatibilidad hasta la versión 7.4 de PHP, con versiones superiores puede provocar algunos errores.

**Reply by locurandres** [reply]
la versión q me da el servidor es la php 7.4.27 si o si tiene q ser  7.4?
![error02](https://github.com/chamilo/chamilo-lms/assets/74663717/828b46fe-4824-4dbc-9772-0b939d1ff6dd)

**Reply by locurandres** [reply]
alguna solución alternativa?

**Reply by jloguercio** [reply]
Apartir de la versión 7.4 de PHP la sintaxis que usa los curly bracers está deprecada :

`$test{0}`

esto debería traducirse a 

`$test[0]`

como en la imagen de abajo (que es de la librería PHPExcel la linea donde da el error)

![Captura](https://github.com/chamilo/chamilo-lms/assets/12191804/f4e6c212-53ee-4362-a693-a2b47d27696c)


Es más un problema de la librería de PHPExcel.

**Reply by locurandres** [reply]
ok, osea q  hay que reemplazar esa linea en que archivo en particular? o hay q revisar toda la librería?

**Reply by locurandres** [reply]
Sos un genio!!! solucionado muchas gracias

**Comment by nosolored** [comment]
¿Cuál es tu versión de Chamilo?
Nosotros tenemos plataformas de Chamilo con la 1.11.26 y PHP 7.4.4 y no nos produce ese error.
Quizá debas actualizar tanto Chamilo como sus dependencias.
